app/models/post.rb

```diff
@@ -1,9 +1,7 @@
 class Post < ApplicationRecord
   belongs_to :user
 
-  def user_email
-    user.email
-  end
+  delegate :email, :to => :user, :allow_nil => true
```

Comment:
I'm assuming this does the same thing as your `user_email` method? How, what & who?

Reply:
Very clever!